GVRf/Framework/jni/shaders/material/unlit_shader.cpp

```diff
@@ -84,6 +84,11 @@ void UnlitShader::render(const glm::mat4& mvp_matrix,
         std::string error = "UnlitShader::render : texture with wrong target";
         throw error;
     }
+    if (texture->getId() == 0)
+    {
+        std::string error = "UnlitShader::render : texture with invalid Id";
+        throw error;
```

Comment:
Doesn't this crash the app? That's what happens when other JNI code throws exceptions ....